### PR TITLE
Edit test script woocommerce

### PR DESCRIPTION
**Pull request by DaisukiDaYo**
Changes proposed in this pull request:
- Modified test script woocommerce plugin
- This is for NPCC https://www.npccny.org/ specifically

### CODE REVIEW

File: woocommerce-puppeteer/woocommerce.test.js [hunk 1]
```diff
@@ -64,13 +75,26 @@ describe('Open site...', () => {
...
     await page.click('#wp-submit')
-    await page.waitForNavigation({waitUntil: 'load'})
+    await page.waitFor(3000)
+
+    const ids = await page.evaluate(() => {
+      const spans = Array.from(document.querySelectorAll('#the-list tr span.id'))
+      return spans.map(span => span.innerHTML)
+    })
+
+    const id = ids.map( id => {
```
Comment (zkan) [inline woocommerce-puppeteer/woocommerce.test.js]: Let's remove the space before `id` in `map` function.

File: woocommerce-puppeteer/woocommerce.test.js [hunk 2]
```diff
@@ -64,13 +75,26 @@ describe('Open site...', () => {
...
+      return id.replace('ID: ', '').replace('|', '').trim()
+    })
+    const lastId = id[0]
+
+    await page.goto(url + '/wp-admin/post.php?post=' + lastId + '&action=edit')
```
Comment (zkan) [inline woocommerce-puppeteer/woocommerce.test.js]: Let's try if we can do something like:

```js
await page.goto(`${url}/wp-admin/post.php?post=${lastId}&action=edit`)
```

Please change everywhere. The code looks nicer than using string concatenation.

Comment (DaisukiDaYo) [reply woocommerce-puppeteer/woocommerce.test.js]: Done.

File: woocommerce-puppeteer/woocommerce.test.js (hunk 1 above)
Comment (DaisukiDaYo) [reply woocommerce-puppeteer/woocommerce.test.js]: Done.

**Pull request merged**